src/types/Constants.ts

```diff
@@ -46,6 +46,7 @@ export enum PAGES {
   PARTNER_NETWORK = 'partnernetwork',
   USER_MANAGEMENT = 'usermanagement',
   TECHNICAL_SETUP = 'technicalsetup',
+  CONNECTOR_MANAGEMENT = 'connectormanagement',
```

Comment:
@lavanya-bmw in line 207 we had the config
 TECHNICAL_SETUP_VIEW = 'view_technical_setup',

This was handling the page access for this page via permission. Now by adding line 49 and changing the page name in the config file, will this still work?

Reply:
@jjeroch I am using the same role and only changed the page name, it still works
{
    name: PAGES.CONNECTOR_MANAGEMENT,
    role: ROLES.TECHNICAL_SETUP_VIEW,
    element: <EdcConnector />,
  },